src/main/java/seedu/address/logic/commands/BlacklistCommand.java

```diff
@@ -0,0 +1,95 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.ClientStatus;
+import seedu.address.model.person.Person;
+
+/**
+ * Blacklists an existing person.
+ */
+public class BlacklistCommand extends Command {
+
+    public static final String COMMAND_WORD = "blacklist";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + " index : adds person to the blacklist";
+
+    public static final String MESSAGE_BLACKLIST_PERSON_SUCCESS = "Blacklisted Person: %1$s";
+
+    private final Index index;
+
+    /**
+     * @param index of the person in the filtered person list to blacklist
+     */
+    public BlacklistCommand(Index index) {
+        requireNonNull(index);
+        this.index = index;
+    }
+
+    /**
+     * Creates a blacklisted copy of the original person.
+     *
+     * @param personToBlacklist the original person to be blacklisted
+     * @return a blacklisted copy of the person
+     */
+    public Person createBlacklistedPerson(Person personToBlacklist) {
```

Comment:
LGTM! Although, could it have just been easier to modify the clientstatus to include "blacklisted", and just use the edit command?

Reply:
Yeah but the edit command is used when you want to edit one or more params; here since we are only editing one specific param its easier to just modify that directly and use the original person's other params.